### PR TITLE
Documentation update after downscaler subchart removal

### PR DESCRIPTION
**Pull request by ArtisKrumins**

### CODE REVIEW

File: simple/values.yaml [hunk 1]
```diff
@@ -16,7 +16,8 @@ environmentName: ""
 app: simple
 
 silta-release:
-  # Enable downscaler by default. See https://github.com/wunderio/charts/blob/master/silta-downscaler/
+  # Enable downscaler by default. See downscaler section
```
Comment (Jancis) [inline simple/values.yaml:19]: Those value files are not related in a functional way. Only default downscaling rules are to be noted there. Functionally, charts only add annotations when enabled, the rest is done by downscaler that is bundled in silta-cluster chart. You can't influence or override settings in that subchart.

**Review by Jancis — CHANGES_REQUESTED** [review]
you might as well remove links to that subchart. developers can only influence schedules and if it's enabled at all.

**Comment by ArtisKrumins** [comment]
> you might as well remove links to that subchart. developers can only influence schedules and if it's enabled at all. and i did not see changes to frontend chart either.

Frontend chart does not have comment or links regarding downscaler

File: simple/values.yaml (hunk 1 above)
Comment (ArtisKrumins) [reply simple/values.yaml]: [Removing reference to silta-cluster chart.](https://github.com/wunderio/charts/pull/397/commits/60d8a5637a045fa93d4aa7d9ccc0d6db6c317610)
This indeed could mislead what is possible to change in release.

Comment (Jancis) [reply simple/values.yaml]: they can, however, customize downscaling delay: https://github.com/wunderio/drupal-project-k8s/pull/632/files

Comment (ArtisKrumins) [reply simple/values.yaml]: Added back, however I still would like to give exact example (and therefor commented) to not confuse with values from silta-cluster chart which expects different type of this value.

Comment (Jancis) [reply simple/values.yaml]: Those example values should be enough. This is basically all developers care about. 
And if possible, keep the `""` definition in configuration too as it will break template condition otherwise (it needs a check that variable is defined otherwise) - https://github.com/wunderio/charts/blob/master/drupal/templates/drupal-ingress.yaml#L79C26-L79C26

Comment (ArtisKrumins) [reply simple/values.yaml]: Good point! Added.

Comment (Jancis) [reply simple/values.yaml]: I know You must hate me by now, but if it would be possible to reduce the number of lines there, it would be awesome :D Adding examples to the first line might do.
Thanks for cleaning up the links :pray:

**Review by Jancis — APPROVED** [review]
